airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -133,6 +136,10 @@ def read(
 
         logger.info(f"Finished syncing {self.name}")
 
+    @property
+    def per_stream_state_enabled(self):
```

Comment:
Unused at the moment, but we'll want this defined so we can parallelize the next two tasks that use this function